TL improvement (#558)
07th-mod/higurashi-console-arcs: Update/omot_014.txt

//r他のテーブルで飲食している客も、この光景には度肝を抜かれたような顔でこっちを見ている。…というか、俺だって十分圧倒されているので、そんな奇異な視線を俺に向けないでもらいたいものだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "他のテーブルで飲食している客も、この光景には度肝を抜かれたような顔でこっちを見ている。…というか、俺だって十分圧倒されているので、そんな奇異な視線を俺に向けないでもらいたいものだ。",

> 			NULL, "The customers eating and drinking at the other tables looked awestruck as they watched the scene unfold. ...I mean, I'm totally overwhelmed too, so I want them to stop suspiciously staring at me.", Line_Normal);

Reviewer: ```suggestion
			NULL, "The customers eating and drinking at the other tables looked awestruck as they watched the scene unfold. ...I mean, I'm overwhelmed enough as it is, so I'd like them to stop suspiciously staring at me.", Line_Normal);
```

//rでも…でも、やっぱりだめ。あんなふうに、二人で楽しく過ごしているのを見たら。そして、あんなふうに庇ってもらえるのを知ったら、…やっぱり悔しい。憎い。どうしようもない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "でも…でも、やっぱりだめ。あんなふうに、二人で楽しく過ごしているのを見たら。そして、あんなふうに庇ってもらえるのを知ったら、…やっぱり悔しい。憎い。どうしようもない。",

> 			NULL, "But... but in the end, it was impossible. Seeing the two of them having fun together, and learning that a girl's boyfriend protects her like that... made me utterly mortifying. I hated it. I couldn't help but feel that way.", Line_Normal);

Reviewer: ```suggestion
			NULL, "But... but in the end, it was impossible. Seeing the two of them having fun together, and learning that a girl's boyfriend protects her like that... made me utterly mortified. I hated it. I couldn't help but feel that way.", Line_Normal);
```

//r真相はさておいても、殺された動機が全く一貫していない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "真相はさておいても、殺された動機が全く一貫していない。",
			NULL, "Whatever the truth was, the motives behind the murders actually weren't consistent at all.", Line_Normal);
	ClearMessage();

//r殺される対象となった理由が、『村にとっての敵』と一致しているだけだ。しかも１年前の主婦は、厳密にはこの枠で扱うのも微妙なほど、村にとってはどうでもいい存在だった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "殺される対象となった理由が、『村にとっての敵』と一致しているだけだ。しかも１年前の主婦は、厳密にはこの枠で扱うのも微妙なほど、村にとってはどうでもいい存在だった。",

> 			NULL, "The one thing the murders had in common was the victims being \"an enemy of the village.\" And strictly speaking, it was doubtful if this logic applies to the housewife's case because she didn't really matter to the village.", Line_Normal);

Reviewer: ```suggestion
			NULL, "The one thing the murders had in common was the victims being \"an enemy of the village.\" And strictly speaking, it was doubtful if that logic even applied in the housewife's case, since she didn't really matter to the village.", Line_Normal);
```
